### ISSUE TITLE
can't connect micro:bit

### ISSUE THREAD

**Issue by lasiaz** [LLK/scratch-vm#1681]
### Expected Behavior

connection micro:bit  

### Actual Behavior

show error modal

### Steps to Reproduce

1. click micro:bit extension
2. show connection modal
3. click connect
4. show error modal

this is the error 
error: {code: -32500, message: "Application Error",…}
code: -32500
data: "Unhandled error encountered during call: System.TypeLoadException: 找不到 Windows 运行时类型“Windows.Devices.Bluetooth.GenericAttributeProfile.GattDeviceServicesResult”。
↵   在 scratch_link.BLESession.Connect(JObject parameters)
↵   在 scratch_link.BLESession.<DidReceiveCall>d__15.MoveNext() 位置 C:\GitHub.lxss\scratch-link\Windows\scratch-link\BLESession.cs:行号 135
↵   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
↵   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
↵   在 scratch_link.Session.<DidReceiveRequest>d__17.MoveNext() 位置 C:\GitHub.lxss\scratch-link\Windows\scratch-link\Session.cs:行号 248
↵   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
↵   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
↵   在 scratch_link.Session.<DidReceiveMessage>d__16.MoveNext() 位置 C:\GitHub.lxss\scratch-link\Windows\scratch-link\Session.cs:行号 205"
message: "Application Error"
id: 1
jsonrpc: "2.0"

### Operating System and Browser

Windows 10  Chrome

**Comment by benjiwheeler** [comment]
Just checking, are you installing and running the Scratch Link Windows application first?

**Comment by lasiaz** [comment]
Yes，can find micro:bit, but can’t connect.

**Comment by cwillisf** [comment]
@lasiaz, which version of Windows 10 do you have? Maybe 10.0.14393 or 10.0.15063?

**Comment by evhan55** [comment]
Is this related to https://github.com/LLK/scratch-link/issues/97?

**Comment by cwillisf** [comment]
I'm starting to think this might be due to the issue discussed here: https://stackoverflow.com/a/43673775

I think it's likely that LLK/scratch-link#97 is this same issue.

TL;DR, if I understand correctly: there's a bug, or at least an inconsistency, with the way that Bluetooth security works in desktop vs. WinRT/WPF-style applications. Since Scratch Link is written in C# it tries to use the WinRT method, but since Scratch Link is actually a desktop app it needs to use the desktop approach. I believe I can do this by using code similar to what's posted at the Stack Overflow link above. We'll need to triple-check that this doesn't break on newer versions of Windows 10, though.
